packages/contractkit/src/wrappers/Election.ts

```diff
@@ -0,0 +1,220 @@
+    this.contract.methods.numberValidatorsInCurrentSet,
+    undefined,
+    toNumber
+  )
+
+  getGroupsVotedFor: (account: Address) => Promise<Address[]> = proxyCall(
+    this.contract.methods.getGroupsVotedFor
+  )
+
+  /**
+   * Returns current configuration parameters.
+   */
+  async getConfig(): Promise<ElectionConfig> {
+    const res = await Promise.all([
+      this.minElectableValidators(),
+      this.maxElectableValidators(),
+      this.electabilityThreshold(),
+      this.contract.methods.maxNumGroupsVotedFor().call(),
+    ])
+    return {
+      minElectableValidators: res[0],
+      maxElectableValidators: res[1],
+      electabilityThreshold: res[2],
+      maxNumGroupsVotedFor: toBigNumber(res[3]),
+    }
+  }
+
+  async getValidatorSetAddresses(): Promise<string[]> {
+    const numberValidators = await this.numberValidatorsInCurrentSet()
+
+    const validatorAddressPromises = []
+
+    for (let i = 0; i < numberValidators; i++) {
+      validatorAddressPromises.push(this.validatorAddressFromCurrentSet(i))
+    }
+
+    return Promise.all(validatorAddressPromises)
+  }
+
+  async getValidatorGroupsVotes(): Promise<ValidatorGroupVote[]> {
+    const validators = await this.kit.contracts.getValidators()
+    const vgAddresses = (await validators.getRegisteredValidatorGroups()).map((g) => g.address)
+    const vgVotes = await Promise.all(
+      vgAddresses.map((g) => this.contract.methods.getGroupTotalVotes(g).call())
+    )
+    const vgEligible = await Promise.all(
+      vgAddresses.map((g) => this.contract.methods.getGroupEligibility(g).call())
+    )
+    return vgAddresses.map((a, i) => ({
+      address: a,
+      votes: toBigNumber(vgVotes[i]),
+      eligible: vgEligible[i],
+    }))
+  }
+
+  async getEligibleValidatorGroupsVotes(): Promise<ValidatorGroupVote[]> {
+    const res = await this.contract.methods.getEligibleValidatorGroupsVoteTotals().call()
+    return zip((a, b) => ({ address: a, votes: new BigNumber(b), eligible: true }), res[0], res[1])
+  }
+
+  /*
+  async revokeVote(): Promise<CeloTransactionObject<boolean>> {
+    if (this.kit.defaultAccount == null) {
+      throw new Error(`missing from at new ValdidatorUtils()`)
+    }
+
+    const lockedGold = await this.kit.contracts.getLockedGold()
+    const votingDetails = await lockedGold.getVotingDetails(this.kit.defaultAccount)
+    const votedGroup = await this.getVoteFrom(votingDetails.accountAddress)
+
+    if (votedGroup == null) {
+      throw new Error(`Not current vote for ${this.kit.defaultAccount}`)
+    }
+
+    const { lesser, greater } = await this.findLesserAndGreaterAfterVote(
+      votedGroup,
+      votingDetails.weight.negated()
+    )
+
+    return wrapSend(this.kit, this.contract.methods.revokeVote(lesser, greater))
+  }
+  */
+
+  async markGroupEligible(validatorGroup: Address): Promise<CeloTransactionObject<boolean>> {
+    if (this.kit.defaultAccount == null) {
+      throw new Error(`missing from at new ValdidatorUtils()`)
```

Comment:
Not immediately clear to me what `ValidatorUtils()` is referring to?

Reply:
it should be `missing kit.defaultAccount`. But even better, we should make this method receive the `from` as an argument